sentry-android-core/src/main/java/io/sentry/android/core/ActivityFramesTracker.java

```diff
@@ -76,7 +96,13 @@ private void snapshotFrameCountsAtStart(final @NotNull Activity activity) {
     if (frameMetricsAggregator == null) {
       return null;
     }
-    final @Nullable SparseIntArray[] framesRates = frameMetricsAggregator.getMetrics();
+
+    @Nullable SparseIntArray[] framesRates = null;
+    try {
+      framesRates = frameMetricsAggregator.getMetrics();
```

Comment:
I agree, I still added it in order to make all `frameMetricsAggregator.x` calls safer. Yes, it's safe now but in theory it could change with a future update to the androidx lib. But I'm fine with removing it 👍

Reply:
Your call, upgrades to the androidx lib can cause new issues, I agree.